### PR TITLE
Adds animation to the Jetpack Powered banner

### PR DESCRIPTION
**Pull request by Gio2018**
Fixes #NA

This PR adds animation to the Jetpack Powered banner. The banner will hide when scrolling down, and show when scrolling back up.

**NOTE: this PR only adds the animation on Stats. The notifications banner will be addressed in a future PR**

<p align=center>

</p>
To test:

- Build/run this branch
- Go to My Site -> Stats
- Select any tab (Insights, Days, etc)
- Make sure the "Jetpack powered" banner hides when scrolling down, and shows when scrolling up (see animation above)

## Regression Notes
1. Potential unintended areas of impact
Low, this just adds animation to the Jetpack Powered banner

2. What I did to test those areas of impact (or what existing automated tests I relied on)
test on a variety of devices

3. What automated tests I added (or what prevented me from doing so)
none
PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] ~~I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.~~ Not released.

### CODE REVIEW

**Review by sla8c — APPROVED** [review]
LGTM!
Additionally I've tested:
 * phone device portrait and landscape
 * iPad (simulator) portrait and landscape

File: WordPress/Classes/ViewRelated/Jetpack/Branding/Banner/JetpackBannerView.swift [hunk 1]
```diff
@@ -29,3 +26,32 @@ class JetpackBannerView: UIView {
...
+        let isHidden: Bool = input < 0
+
+        guard self.isHidden != isHidden else {
+            return .unlimited
+        }
+
+        UIView.animate(withDuration: 0.3, delay: 0, options: [.curveEaseIn], animations: { [weak self] in
+            self?.isHidden = isHidden
+        }, completion: { _ in
+
```
Comment (alpavanoglu) [inline WordPress/Classes/ViewRelated/Jetpack/Branding/Banner/JetpackBannerView.swift]: Shouldn't we pass nil to completion here?

Comment (Gio2018) [reply WordPress/Classes/ViewRelated/Jetpack/Branding/Banner/JetpackBannerView.swift]: good catch @alpavanoglu ! Indeed we should

**Comment by Gio2018** [comment]
Thanks for reviewing @sla8c @alpavanoglu !

**Pull request merged**